src/librustdoc/clean/mod.rs

```diff
@@ -2682,11 +2668,12 @@ fn name_from_pat(p: &hir::Pat) -> String {
     match p.node {
         PatKind::Wild => "_".to_string(),
         PatKind::Binding(_, ref p, _) => p.node.to_string(),
-        PatKind::TupleStruct(ref p, ..) | PatKind::Path(None, ref p) => path_to_string(p),
+        PatKind::TupleStruct(ref p, ..) | PatKind::Path(None, ref p) => p.to_string(),
```

Comment:
Do you think I should add an parameter to that function to control whether the comment gets printed or just ditch that commit?